src/github.com/getlantern/eventual/eventual_test.go

```diff
@@ -14,6 +15,7 @@ const (
 )
 
 func TestSingle(t *testing.T) {
+	goroutines := runtime.NumGoroutine()
```

Comment:
This way of testing is sensitive to previous tests.  If a previous test spawned a goroutine that ends after this line but before you call `runtime.NumGoroutine()` again, the count will be wrong.